src/ui/controllers/user.ts

```diff
@@ -529,7 +391,7 @@ export async function patchAndUpdate(
 				res.redirect(`/profile/${inputName}`)
 			})
 		}
-	} else if (response) {
+	} else if (requestWasSuccessful) {
 		// seems like we have to get the profile again to get values
```

Comment:
just a question, are we going to do something about this unnecessary call to get `profile`?

Reply:
it is currently necessary as the patch method returns the civilServant in the HATEOAS format